app/controllers/developers/celebration_package_requests_controller.rb

```diff
@@ -1,14 +1,15 @@
-module Hired
-  class FormsController < ApplicationController
+module Developers
+  class CelebrationPackageRequestsController < ApplicationController
+    before_action :test
     before_action :authenticate_user!
     before_action :require_developer!
 
     def new
-      @form = Hired::Form.new(developer: current_user.developer)
+      @form = Developers::CelebrationPackageRequest.new(developer: current_user.developer)
```

Comment:
I think we can remove the `Developers::` namespace since we are already in the module.

Reply:
Removed in 657afa99c63b8ea99aab168a1677fda9cc2596d2